app/assets/stylesheets/_authors.scss

```diff
@@ -0,0 +1,9 @@
+.author__name{
+  font-size:1.5em;
+}
+
+.author__books{
+  @include display(flex);
+  @include border-color(null null #CCCCCC null);
```

Comment:
Color literals like `#CCCCCC` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#CCCCCC` should be written as `#cccccc`